onnx/defs/quantization/defs.cc

```diff
@@ -42,7 +50,7 @@ ONNX_OPERATOR_SET_SCHEMA(
             "T2",
             {"tensor(int8)", "tensor(uint8)"},
             "Constrain 'y_zero_point' and 'y' to 8-bit integer tensor.")
-        .SetDoc(QuantizeLinear_ver10_doc)
+        .SetDoc(QuantizeLinear_ver13_doc)
         .TypeAndShapeInferenceFunction(
             [](ONNX_NAMESPACE::InferenceContext& ctx) {
               propagateElemTypeFromInputToOutput(ctx, 2, 0);
```

Comment:
There is a bug here. y_zero_point is optional. if it doesn't exist, we should set the output type uint8.

Reply:
I've opened https://github.com/onnx/onnx/issues/2839 to track this issue, we can resolve this in another MR.